src/actions/index.js

```diff
@@ -1,8 +1,23 @@
 // we'll need axios
-
+import axios from 'axios';
 // we'll need to create 3 different action types here.
 // one for fetching, one for success and one for failure
+export const FETCH = 'FETCH';
+export const SUCCESS = 'SUCCESS';
+export const FAILURE = 'FAILURE';
 
 // our action creator will be a function that returns a function
 // the url to fetch characters from is `https://swapi.co/api/people/`
 // remember that now we have controll over our thunk-based action creator
+
+export const getData = () => dispatch => {
+  dispatch({type: FETCH})
+  axios
+    .get('https://swapi.co/api/people/')
+    .then(response => {console.log(response); 
+      dispatch({ type: FETCH, payload: response.data.results});}
```

Comment:
You have declared an action type of `SUCCESS` above, and you have a switch condition for it in your reducer, but you never actually dispatch an action with a type of `SUCCESS`. That is why your app does not work. 

If you change: 
`dispatch({ type: FETCH, payload: response.data.results});}` 
to: 
`dispatch({ type: SUCCESS, payload: response.data.results});}`

then your application works correctly.